blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/horizontal-links/default.jsx

```diff
@@ -51,7 +40,7 @@ const HorizontalLinksBar = ({ hierarchy, navBarColor, showHorizontalSeperatorDot
 	const readableContrastingColor = navBarColor === "light" ? "#000" : "#fff";
 
 	return (
-		<ReadableTextNavigationBar
+		<nav
 			key={id}
 			color={readableContrastingColor}
```

Comment:
This is a styled component prop that will need moved to a CSS approach

Reply:
Will be addressed in https://arcpublishing.atlassian.net/browse/TMEDIA-888